### PR TITLE
Give informative error message when passing a relative path

### PR DESCRIPTION
**Pull request by mrocklin**
Any suggestions on the right way to trigger this for a test?

### CODE REVIEW

**Comment by mrocklin** [comment]
Also, is this appropriate?

**Comment by mrocklin** [comment]
Alternatively, what is the argument against doing this for them?

File: skein/model.py [hunk 1]
```diff
@@ -507,7 +507,11 @@ def _normpath(path, origin=None):
                 if origin is not None:
                     path = os.path.normpath(os.path.join(origin, url.path))
                 else:
-                    raise ValueError("paths must be absolute")
+                    msg = ("Environment paths must be absolute. \n"
```
Comment (jcrist) [inline skein/model.py]: Paths, not environment paths. Path could be any archive or file.

**Comment by jcrist** [comment]
I decided it made sense to support them (nicities outweigh the footguns), added in #29. Apologies for pushing a separate PR, I'm trying to get a tiny bugfix release out.

**Comment by mrocklin** [comment]
No apologies necessary for making it so that I don't have to do more work :)

**Pull request closed without merging**